examples/transfer-files/public/js/app.js

```diff
@@ -114,6 +114,10 @@ function onDrop (event) {
   onDragExit()
   $errors.className = 'hidden'
   event.preventDefault()
+  if (!node) {
+    onError("IPFS must be started before files can be added")
```

Comment:
This is causing a `linting` error, could you change to single quotes, please?

https://travis-ci.org/ipfs/js-ipfs/jobs/253836140#L4278